src/qml/Mysthea/Components/AppContainer.qml

```diff
@@ -0,0 +1,15 @@
+import QtQuick 2.11
```

Comment:
It is better to import the latest version of  `QtQuick` and `QtQuick.Controls`. Each of them are at 2.14.